hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -2083,6 +2092,11 @@ private void listStatusFindKeyInTableCache(
       metadataManager.getLock().releaseReadLock(BUCKET_LOCK, volumeName,
           bucketName);
     }
+    if (args.getRefreshPipeline()) {
+      for(OzoneFileStatus fileStatus : fileStatusList){
+        refreshPipeline(fileStatus.getKeyInfo());
```

Comment:
I'm +1 on this JIRA. 

Can we file a follow up JIRA to allow batching refreshPipeline call instead of individual calls to save RPC traffic.

Reply:
Thanks @xiaoyuyao , I have raised HDDS-3961 to handle this case.